files/common/usr/bin/download-latest-image

```diff
@@ -69,8 +79,8 @@ aws s3 cp \
 	"s3://snapshot-de-images/builds/jenkins-ops/devops-gate/master/appliance-build/$opt_b/$build/latest" \
 	. || die "failed to download file: 'latest'"
 
-$opt_f && rm -f "$1.upgrade.tar" >/dev/null 2>&1
-[[ -f "$1.upgrade.tar" ]] && die "image $1.upgrade.tar already exists"
+$opt_f && rm -f "$VARIANT.upgrade.tar" >/dev/null 2>&1
+[[ -f "$VARIANT.upgrade.tar" ]] && die "image $VARIANT.upgrade.tar already exists"
 
-aws s3 cp "s3://snapshot-de-images/$(cat latest)/upgrade-artifacts/$1.upgrade.tar" . ||
-	die "failed to download file: '$1.upgrade.tar'"
+aws s3 cp "s3://snapshot-de-images/$(cat latest)/upgrade-artifacts/$VARIANT.upgrade.tar" . ||
```

Comment:
nit: might be useful to echo the full S3 path for debugging/informational purposes.

Reply:
I disagree.